pkg/ddl/foreign_key.go

```diff
@@ -206,19 +206,11 @@ func checkTableForeignKeyValid(is infoschema.InfoSchema, schema string, tbInfo *
 	return checkTableForeignKey(referTblInfo, tbInfo, fk)
 }
 
-func getAndCheckLatestInfoSchema(d *ddlCtx, _ *meta.Meta) (infoschema.InfoSchema, error) {
-	// TODO(crazycs520): fix me, need to make sure the `d.infoCache` is the latest infoschema.
```

Comment:
As the comment said, I think this function may be updated to add a checking logic in future. Because you deleted it, do we solve this problem now?

Reply:
same reason as in description of https://github.com/pingcap/tidb/pull/53882, i think we don't need to check it

there is a check before, but deleted https://github.com/pingcap/tidb/pull/38071/files#diff-2218b0b129e90b1e049edf99a3a1c5192249feadb02f54a00b426bf14ac9d71dR189 at 2 years ago